blogs/2020/06/09/go-extension.md

```diff
@@ -10,9 +10,9 @@ Author: Alessandro Segala
 
 June 9, 2020 by The VS Code Team, [@code](https://twitter.com/code)
 
-Since the very beginning, we have worked with developer communities to build Visual Studio Code as a powerful, extensible editor. Five years into this journey, individuals and organizations in our community have helped us create an editor that is really for all developers, using any language.
+From the beginning, we have worked with developer communities to build Visual Studio Code as a powerful, extensible editor. Five years into this journey, individuals and organizations in our community have helped us create an editor that is really for all developers, using any language. VS Code is now the primary editor for many of our 11 million users.
```

Comment:
How about: 
"VS Code is now the primary editor for many of the 11 million Go users."

Reply:
Actually that is covered in the next paragraph. Do we even need this sentence?

Reply:
No, because 11 million is VS Code users all-up, not just Go!

Reply:
I see of the 11 million VS Code users, many use VS Code as their primary editor.